account_credit_control/migrations/13.0.2.0.0/post-migration.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020 Tecnativa - Jairo Llopis
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    # Update mail.template records for credit.control.communication
```

Comment:
This have to check if you have already these objects filled, and don't execute, as if you are coming from 12.0.3.0.0, this will fail.